### PR TITLE
Support wsteth complex path, fix other tests

### PR DESCRIPTION
**Pull request by talbaneth**

### CODE REVIEW

File: src/test/WstETHCurveUniv3Callee.t.sol [hunk 1]
```diff
@@ -153,40 +159,42 @@ contract CurveCalleeTest is DSTest {
...
         uint256 amt = 3000 * WAD;
         newAuction(amt);
+        uint24 pooAlFee = 500;
+        uint24 pooBlFee = 100;
         bytes memory data = abi.encode(
             address(this),
             address(gemJoin),
             uint256(0),
-            uint24(3000),
+            abi.encodePacked(weth, pooAlFee, usdc, pooBlFee, dai),
```
Comment (julienmartinlevrai) [inline src/test/WstETHCurveUniv3Callee.t.sol:175]:
```suggestion
            abi.encodePacked(weth, poolAFee, usdc, poolBFee, dai),
```

**Review by julienmartinlevrai — APPROVED** [review]
LGTM

**Review by godsflaw — APPROVED** [review]
LGTM

**Review by julienmartinlevrai — APPROVED** [review]
Etherscan-verified source code of deployed contract matches locally flattened repo code

**Review by godsflaw — APPROVED** [review]
address matches code

**Pull request merged**